enterprise/cli/workspaceproxy.go

```diff
@@ -260,6 +296,11 @@ func (r *RootCmd) createProxy() *clibase.Cmd {
 			Description: "Display icon of the proxy.",
 			Value:       clibase.StringOf(&proxyIcon),
 		},
+		clibase.Option{
+			Flag:        "no-prompts",
```

Comment:
Suggestion: Singular feels more consistent with out CLI. This could potentially be a global flag for clibase too.

```suggestion
			Flag:        "no-prompt",
```